src/App.js

```diff
@@ -1,83 +1,68 @@
-import React from "react";
-import "./App.css";
+import React, { useEffect, useState } from 'react';
+import './App.css';
+import { getFilteredForecasts } from './utils/utils';
+import { cities } from './utils/cities.js';
+
+import CurrentWeather from './components/CurrentWeather/CurrentWeather';
+import Forecast from './components/Forecast/Forecast';
+import Spinner from './components/Spinner/Spinner';
 
 const App = () => {
+  const [weather, setWeather] = useState(null);
+  const [city, setCity] = useState('Prague');
+  const [forecasts, setForecasts] = useState(null);
+
+  useEffect(() => {
+    const API_ID = process.env.REACT_APP_MY_API_ID;
```

Comment:
API_ID a funkce fetch jsem přesunula přímo do useEffectu, protože [cities] bylo podtržené a hlásilo chybu 'react hook useEffect has missing dependencies' a tohle jsem našla na stackOverflow, ale úplně tomu nerozumím, tak pokud bys mi k tomu mohla napsat víc, budu ráda :)

Reply:
Nakonec jsem to vrátila zpět jako pokus o vyřešení problému s netlify, nicméně to nepomohlo.